test/skimNtuple_HHbtag.cpp

```diff
@@ -2959,28 +2957,32 @@ int main (int argc, char** argv)
 				  float lep1_thresh = trgAssigner.getLepton1PtThresh(PERIOD, "mutau");
 				  float lep2_thresh = trgAssigner.getLepton2PtThresh(PERIOD, "mutau");
 				  if (tlv_firstLepton.Pt()  < lep1_thresh) passSingle = 0;
```

Comment:
I am a bit confused by these passSingle and passCross lines Bruno, sorry. At this point in the code the trigger matching has already happened and we are simply retrieving scale factors right? I think these two flags should be set when the trigger matching is performed no?

Reply:
these have always been here, no? in order to properly combine the object-level efficiencies of the single-lepton & cross triggers

Reply:
Ah yes, they were, my bad, sorry. It became a bit difficult to remember what was there and what not eheh